donate.html

```diff
@@ -12,6 +12,70 @@
         <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+    <script type="text/javascript">
+        	function hider(channel,no) {
+    // var x = document.getElementById('myDIV');
+    // if (x.style.display === 'none') {
+    //     x.style.display = 'block';
+    // } else {
+    //     x.style.display = 'none';
+    // }
+    console.log("Asdf");
+    if(channel==1)
+   {
+
+    var x = document.getElementById('donation1')
+    var y = document.getElementById('regular1');
+    if(x.style.display==='none')
+    {
+    if(no==1)
+    {
+     y.style.display = 'none';	
+    
+    x.style.display = 'block';
+    }
+   
+   	}
+   	if(y.style.display==='none')
+   	{
+    if(no==2)
+    { 
+    y.style.display = 'block';	
+	x.style.display = 'none';
+ 	
+    }
+   	}
+
+}
+ if(channel==2)
+   {
+
+    var x = document.getElementById('donation2')
+    var y = document.getElementById('regular2');
+    if(x.style.display==='none')
+    {
+    if(no==1)
+    {
+     y.style.display = 'none';	
```

Comment:
@abhishek1995s Great work for caching `document.getElementById('donation2')` it seems that we are just doing the same stuff of `display: none` and `display: block` again and again. I would suggest making a CSS class say `appear { display: block;} `   and `disappear {display: none}` and add/remove them through JS